src/MagnumPlugins/AnyImageImporter/AnyImageImporter.cpp

```diff
@@ -128,8 +130,11 @@ void AnyImageImporter::doOpenData(Containers::ArrayView<const char> data) {
     CORRADE_INTERNAL_ASSERT(manager());
 
     std::string plugin;
+    /* https://github.com/BinomialLLC/basis_universal/blob/7d784c728844c007d8c95d63231f7adcc0f65364/transcoder/basisu_file_headers.h#L78 */
+    if(Utility::String::viewBeginsWith(data, "sB\0\x16")
+        plugin = "BasisImporter";
     /* https://docs.microsoft.com/cs-cz/windows/desktop/direct3ddds/dx-graphics-dds-pguide */
-    if(Utility::String::viewBeginsWith(data, "DDS "))
+    else if(Utility::String::viewBeginsWith(data, "DDS "))
```

Comment:
Care to add a test for this to the `detect()` test case? It's really simple, just pick the smallest basis file you have (don't forget to reference it in CMakeLists) and then add one line to the test instance data. It'll auto-check both the extension and magic comparison.